src/Microsoft.Azure.WebJobs.Host/Filters/IFunctionInvocationFilter.cs

```diff
@@ -4,23 +4,24 @@
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.Azure.WebJobs.Host
+namespace Microsoft.Azure.WebJobs.Host.Filters
```

Comment:
>Filters [](start = 39, length = 7)

I'd vote against the sub namespace here. Function authors will be directly consuming this interface and itd be good to be top-level to be more discoverable.